plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/everytrade/EveryTradeBeanV1.java

```diff
@@ -97,9 +97,13 @@ public TransactionCluster toTransactionCluster() {
                 )
             );
         }
-        return new TransactionCluster(
+        TransactionCluster transactionCluster = new TransactionCluster(
```

Comment:
make it `var` and to one line as `var transactionCluster = new TransactionCluster(buySell, related);`